### DISCUSSION TITLE
Is the Cytoscape desktop application (written in Java) implemented based on cytoscape.js?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [cytoscape/cytoscape.js#3295] [Q&A]
Excuse me, I have 2 questions about Cytoscape:

Is the [Cytoscape desktop application (written in Java)](https://cytoscape.org/releasenotes.html) implemented based on [cytoscape.js](https://github.com/cytoscape/cytoscape.js)?

Is the Cytoscape desktop application (written in Java) open source? Here on GitHub, I see: [cytoscape-impl](https://github.com/cytoscape/cytoscape-impl). I am not sure if this project is the repo for this application.

**Comment by mikekucera** [comment]
Hi,
No, Cytoscape Desktop (Java) is not based on cytoscape.js. 
Both Cytoscape Desktop and Cytoscape.js are open source.

This is not the usual place people ask questions about cytoscape desktop. Please post your questions to the cytoscape-helpdesk google group: https://groups.google.com/g/cytoscape-helpdesk

**Reply by maxkfranz** [reply]
N.b. Cytoscape Java is LGPL, not MIT like JS
